waku/v2/protocol/lightpush/waku_lightpush_test.go

```diff
@@ -336,7 +336,7 @@ func TestWakuLightPushCornerCases(t *testing.T) {
 
 	// Test corner case with default pubSub topic
 	_, err = client.Publish(ctx, msg2, WithDefaultPubsubTopic(), WithPeer(host2.ID()))
-	require.NoError(t, err)
+	require.Error(t, err)
```

Comment:
I'm wondering if this is the right fix for this test?

Reply:
I guess a way to make sure the test is correct could be to check if the error emitted is really a `errors.New("cannot publish to unsubscribed topic")`?